assets/js/base/components/cart-checkout/product-price/style.scss

```diff
@@ -3,3 +3,7 @@
 		margin-left: 0.5em;
 	}
 }
+.wc-block-components-product-price__regular {
+	text-decoration: line-through;
+	opacity: 0.6;
```

Comment:
I tested some shades and 0.6 was the closest to the designs.

Reply:
I don't think we should force a specific opacity, that should be handled by the theme. Also, notice adding `opacity: 0.6` here, makes text not accessible in Storefront with Firefox (`#d2d2d2` over `#ffffff`).

![imatge](https://user-images.githubusercontent.com/3616980/91960405-284fdd80-ed0a-11ea-99b5-17c9803c5c70.png)

---

I see the original issue (`main` branch in Chrome or Safari) is because `opacity` is not inherited from the theme because there is a `<span>` inside a `<del>` and it has `display: block`. I wonder if something like [this](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3065/files#diff-c8f144109dd5ca6eef80b585bbd79a6eR62-R76) would work, so classes are applied directly to the `<del>` tag:

```JSX
<FormattedMonetaryAmount
	renderText={ ( val ) => (
		<del
			className={ classNames(
				'wc-block-components-product-price__regular',
				className
			) }
		>
			{ val }
		</del>
	) }
	currency={ currency }
	value={ regularValue }
/>
```

Doing the change above, I think we can also remove the `text-decoration: line-through;` declaration.

Actually, taking a closer look, all issues fixed in this PR seem to be fixed in #3065 as well. (Too bad I didn't notice when creating the PR :facepalm: ). So we can either:
1. Update this PR as explained above, merge it and rebase #3065.
2. Get #3065 reviewed and if it does fixes the issues, close this PR in favor of that one.